src/core/pick-models.js

```diff
@@ -21,6 +21,10 @@ export default function pickModels(gl, {
   assert(isWebGL(gl), ILLEGAL_ARG);
   assert(framebuffer, ILLEGAL_ARG);
 
+  if (position === null) {
```

Comment:
1) `if (!position)` is usually better, also handles `position === undefined`.

2) Should the `position === null` case be handled inside this function, or outside, by the caller? 

I find it a little surprising that it is handled here, I wouldn't expect this behavior from this function.